core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -2480,6 +2480,27 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     }
   }
 
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("zk", "kraft"))
+  def testAppendConfigToEmptyDefaultValue(ignored: String): Unit = {
+    client = Admin.create(createConfig)
+    val topic = "testAppendConfigToEmptyDefaultValue"
+    createTopic(topic)
```

Comment:
Could we also add a test for the case that topic config already contains a `LeaderReplicationThrottledReplicasProp` value, to make sure the result is what we expected? Ex: 

```java
    val p = new Properties()
    p.setProperty(LogConfig.LeaderReplicationThrottledReplicasProp, "1:1")
    createTopic(topic, topicConfig = p)
...
    assertEquals("1:1, 0:0", config.value())
```